deploy.sh

```diff
@@ -3,23 +3,18 @@
 # To publish a build, use the Makefile
 # make publish
 # which will call out to this script
+set -e
 
 # Use maven-help-plugin to get the current project.version
-VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '\[')
-
-# Tag the new version
-git tag $VERSION
-
-# Push the tag
-git push --tags
+VERSION=`mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '\['`
 
 echo "Publishing $VERSION"
 
 # Build and deploy to Bintray
-mvn deploy
+mvn -s .circleci.settings.xml -Dmaven.test.skip=true deploy
 
 # Sign the jar and other files in Bintray
-curl -H "X-GPG-PASSPHRASE:$BINTRAY_GPG_PASSPHRASE" -u $BINTRAY_USER:$BINTRAY_API_KEY -X POST https://api.bintray.com/gpg/lightstep/maven/otel-launcher-java/versions/$VERSION
+curl -H "X-GPG-PASSPHRASE:$BINTRAY_GPG_PASSPHRASE" -u $BINTRAY_USER:$BINTRAY_API_KEY -X POST "https://api.bintray.com/gpg/lightstep/maven/lightstep-tracer-jre/versions/$VERSION"
```

Comment:
Also, this needs to refer to "otel-launcher-java" (as it was before this PR). Sorry, totally missed this in the previous review ;(

Reply:
fixed